test/services/sync/strategies/CSVSyncer/CSVSyncer.unit.test.js

```diff
@@ -75,5 +78,23 @@ describe('CSVSyncer', () => {
 			expect(result[1].class).to.equal('');
 			expect(result[2].class).to.equal('2b');
 		});
+
+		it('should not accept unvalid dates as user birthday', () => {
+			const falsyValues = [false, null, undefined, '', 0]
+			const misfitValues = [Symbol(), [], {}, true]
+			const wrongFormatDates = ['32.12.2000', '01.13.2000', '01.01-2000', '01/01.2000', '01-01/2000', '42', 'void']
+			const correctFormatDates = ['01.01.2000', '01-01-2000', '01/01/2000']
+
+			const result0 = falsyValues.map(f => CSVSyncer.dateValidator(f))
```

Comment:
Split this into multiple test cases, so if one of these fails, you can see from the name of the failing tests which functionality broke and you don't have to look into the test every time